packages/reassure-cli/src/commands/measure.ts

```diff
@@ -110,6 +114,12 @@ export const command: CommandModule<{}, MeasureOptions> = {
       .option('commit-hash', {
         type: 'string',
         describe: 'Commit hash of current code to be included in the report',
+      })
+      .option('testMatch', {
+        alias: 'tm',
+        type: 'string',
```

Comment:
It seems that Jest accepts an array of globs, could we support that as well. Yargs seems to support array type.
https://jestjs.io/docs/cli#--testmatch-glob1--globn

Reply:
Yes, but we're currently using a single string so I went with the current pattern and added an improvement issue to actually properly update and optimise the glob that we're using as default. Please see https://github.com/callstack/reassure/issues/363

Reply:
I would like to make a separate PR to adjust that and also introduce the `__perf__` directory as it would reflect the Jest defaults better.

Reply:
I'm good with separate PR